api/src/index.ts

```diff
@@ -3,6 +3,13 @@ import express, { Request, Response, NextFunction } from 'express';
 const app  = express();
 const port = 3000;
 
+app.use(function(req, res, next) {
+  res.setHeader('Access-Control-Allow-Origin', 'http://localhost:4200')
```

Comment:
**question:** If you are adding per-environment controls for Angular in this PR, do we need to do something similar for the API to listen on a safe URL? Or is this part of the Express infrastructure that isn't run on Lambda? If so, maybe big, loud comments in this file about what is only part of the application for local development?

Reply:
Trying to reply to this specific question had me realize that while the frontend and backend are connected in that they know how to locate each other locally, this PR does not yet cover deploying Angular at a path through the Express app. This seems like an essential part of this story, so I'll continue working on that.

Reply:
I ended up having to set CORS headers to `'*'` to get the sandbox deployment to work, and I'm not sure why. Setting it to the frontend host (the S3 URL) didn't work. I wonder if this has something to do with the API Gateway, but the resources we're using via Claudia aren't visible to me in the sandbox environment.